### PR TITLE
Fix reshape interoperability test (#17155)

### PR DESCRIPTION
**Pull request by ChaiBapchya**
* fix reshape interoperability test

* fix for scipy import

## Description ##
To fix the numpy.decorator issue in https://github.com/apache/incubator-mxnet/pull/17286

@haojin2 @frankfliu @reminisce

### CODE REVIEW

File: ci/docker/install/docs_requirements [hunk 1]
```diff
@@ -26,11 +26,11 @@ h5py==2.8.0rc1
 mock==2.0.0
 nose==1.3.7
 nose-timer==0.7.3
-numpy>1.16.0,<2.0.0
+numpy>1.16.0,<1.18.0
```
Comment (reminisce) [inline ci/docker/install/docs_requirements]: Why is there a restriction on the numpy version? I think it has been fixed in master.

Comment (ChaiBapchya) [reply ci/docker/install/docs_requirements]: I was told it was this PR's fix that helped resolve the issue https://github.com/apache/incubator-mxnet/issues/17154

**Comment by ChaiBapchya** [comment]
Weird thing

https://github.com/ChaiBapchya/incubator-mxnet/blob/60dd6bd0dc7b902df50b3c8047d13f3c16647050/ci/docker_cache_requirements#L22

On the same branch python-dateutil==2.7.2 has been fixed... not sure why it doesn't pick that up.

**Comment by ChaiBapchya** [comment]
Closing in favor of #17426

**Pull request closed without merging**